### DISCUSSION TITLE
github to gitlab migration is taking very long time

### DISCUSSION THREAD

**Discussion by ssolipurambdc** [community/community#72161] [New to GitHub]
We have few repositories in github and are migrating them to gitlab using the following link:
https://docs.gitlab.com/ee/user/project/import/github.html

Some of the repos are taking very long time to import to gitlab. Around 6+ hours for each repo with commit history, merge requests,issue events.....

When checking the rate limit api, its set to 5000. I think its the hard limit.
We captured the rate limit during the import and its coming down sometimes all the way to zero and recharging again to 5000

Is there a way to speedup the import process from github to gitlab?

**Comment by queenofcorgis** [comment]
👋🏼 @ssolipurambdc, 

Have you checked on the GitLab side of things for this?

**Reply by ssolipurambdc** [reply]
We did and the limits are very high. We previously did a gitlab to gitlab migration and it was fast. Github to gitlab migration is taking long time. Is there anyway to speed up this process
